docs/technical_reference/flowsheets/ADM1.rst

```diff
@@ -0,0 +1,102 @@
+Anaerobic Digestion Model No. 1
+===============================
+
+Introduction
+------------
+
+The IWA created Anaerobic Digestion Model No. 1 with a mathematical model to predict anaerobic digestion scenarios. 
+The objective of the mathematical model is to simulate biological reactions simultaneously with the disintegration, uptake, and decay of microorganisms
+in anaerobic sludge. The property and reaction relationships `ADM1
+<https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ADM1.html>`_
+of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in 
+`Batstone, D. J. et al. (2002) 
+<https://iwaponline.com/wst/article-abstract/45/10/65/6034/The-IWA-Anaerobic-Digestion-Model-No-1-ADM1?redirectedFrom=fulltext>`_ and `Rosen, C. et al. 
+(2006) <https://pubmed.ncbi.nlm.nih.gov/17037165/>`_
+Implementation
+--------------
+
+Figure 1 shows the process flow diagram for ADM1 where the flowsheet only includes the anaerobic digester ehich processes sludge to produce a 
+biogas stream and residual sludge. S_h2 and S_ch4 exist in both vapor phase and liquid phase, S_co2 exists vapor phase, and the other components 
+only exist in the liquid phase. The amount of CO2 dissolved in the liquid phase is equivalent to S_IC - S_HCO3-.
+The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property and reaction packages are provided for the anaerobic digester model (ADM)
+
+.. figure:: ../../_static/flowsheets/ADM1.png
+    :width: 800
+    :align: center
```

Comment:
This image looks really big in the documentation... is there an easy way to shrink it a little bit?